prophesee_ros_driver/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 2.8.3)
 project(prophesee_ros_driver)
 
 ## Compile as C++11, supported in ROS Kinetic and newer
-add_compile_options(-std=c++11)
+add_compile_options(-std=c++14)
```

Comment:
I passed the code rapidly, what did you use of C++14?

Reply:
I need it because of the linux driver package.
/usr/include/prophesee_core/algos/core/internal_algorithms.h:40:55: error: use of ‘auto’ in lambda parameter declaration only available with -std=c++14
I am happy to keep C++11 but it needs some changes in the linux driver

Reply:
Argh... okie! Thanks